### ISSUE TITLE
Can't add existing author to an article

### ISSUE THREAD

**Issue by gemusehandler** [pkp/pkp-lib#2246]
Each journal builds up a database of authors. Some of these are returning (co-)authors. In the current OJS it is not possible to select an author from the database and add him or her to an article. This makes (quick submit-like) processes unnecessary complicated.

**Comment by asmecher** [comment]
Agreed -- though simply providing autocomplete-style searching would potentially expose the author database for abuse. One viable approach might be to permit authors to include co-author ORCIDs, and to have an external confirmation process allow those co-authors to confirm before they're formally added.

**Comment by gemusehandler** [comment]
Why auto-complete searching. In OJS 3 you can add a participant to the article, or a reviewer. Why not an author using the same procedure. The option would especially be useful in a quick submit like process for periodicals with returning authors.

**Comment by asmecher** [comment]
@gemusehandler, those functions are currently available to representatives of the journal, with whom a user creates a "contract" when registering. I'm assuming you'd want to add this process to the submission process, which is available to authors. Authors have no contract with other users of the site, so shouldn't be allowed to arbitrarily search/view information on other users of the system.

**Comment by gemusehandler** [comment]
@asmecher , maybe I haven't played enough with OJS3 yet, but as editor I seem to be able to add a 'contributor' to an article by filling in the usual form, but I can't select an author from the database. Or am I mistaken?

**Comment by asmecher** [comment]
@gemusehandler, OJS tracks "authors" (as in the scholarly metadata) separately from "users" (as in people who interact with the workflow). The "participants" grid allows you to add new users in order to grant them access to the workflow. The "authors" grid allows you to add new contributors to the submission who will be listed in the scholarly metadata.

**Comment by Ph-We** [comment]
I would agree with @gemusehandler, that at least in the case with the Quick Submit that functionality would be both quite useful **and** safe in every respect (since authors/readers do not have access to it).

**Comment by asmecher** [comment]
Ah, that would indeed be safe to add to QuickSubmit. @gemusehandler, is that what you're using?

**Comment by gemusehandler** [comment]
That is exactly what I am asking for...

**Comment by GrazingScientist** [comment]
@asmecher Did you have the time to pursuit this issue further?
